src/index.spec.js

```diff
@@ -498,6 +593,144 @@ describe('Dropzone', () => {
       expect(dropRejectedSpy.callCount).toEqual(0);
     });
 
+    it('accepts files on the innermost Dropzone when we drop into it', () => {
```

Comment:
Yeah, as I already asked ↑ do we really need to support that case? It seems very edgy to me and the complexity it brings to the project doesn't seem worth it. What's the use case for it?